src/material.h

```diff
@@ -75,4 +77,13 @@ class material_type
         const mat_burn_data &burn_data( size_t intensity ) const;
 };
 
+namespace materials
```

Comment:
Ok, but you'll still need to prove both why you need the extra layer of indirection and how having `class material` and `namespace materials` isn't confusing given there's only one character difference?